oidcserverv2/server.go

```diff
@@ -266,7 +398,6 @@ func (s *Server) authorization(w http.ResponseWriter, req *http.Request) {
 	ar, err := s.oidc.StartAuthorization(w, req)
 	if err != nil {
 		l.WithError(err).Error("error starting authorization flow")
-		http.Error(w, "Internal error", http.StatusInternalServerError)
```

Comment:
Not sure I understand this removal. Will the user just see a blank page in the case of this error now?

Reply:
The `StartAuthorization` [handles the HTTP error to the client](https://github.com/pardot/oidc/blob/master/core/oidc.go#L151-L152), so this was superfluous